priv/repo/migrations/20200521031600_create_tasks.exs

```diff
@@ -30,7 +30,7 @@ defmodule BikeBrigade.Repo.Migrations.CreateTasks do
       add :pickup_province_string, :string
       add :pickup_postal, :string
       add :pickup_country, :string
-      add :pickup_loacation, :geography
+      add :pickup_location, :geography
```

Comment:
Let's undo this one, and everything in the `priv/repo/migrations/` folder. 

Migrations are intended to be run sequentially against the database to get it to a known state, so it's bad practice to change past migrations. And this typo in the column name is already fixed below in the other migration